.circleci/config.yml

```diff
@@ -1369,6 +1369,25 @@ jobs:
       - store_test_results:
           path: libs/utils/reports/
 
+  test-crate-ballot-encoder-rs:
```

Comment:
Where are you imagining eventually building the unified ballot interpreter?

I ask because I'm wondering if there's a need for this to live in a separate lib or if it could just live within libs/ballot-interpreter?

Reply:
I could see it being within `libs/ballot-interpreter`. I'll see about consolidating.

Reply:
Consolidated in [3aead87](https://github.com/votingworks/vxsuite/pull/6364/commits/3aead871fd695ac9f016b2c5e605ea6094f03bf5).